views/pages/services/mathsoc-office.pug

```diff
@@ -0,0 +1,34 @@
+extends /views/templates/frontend-base.pug
+include /views/mixins/pink-button.pug
+
+append styles
+    link(href="/assets/css/pages/services/mathsoc-office.css" rel="stylesheet")  
+    link(rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css")
+
+block body 
+    each section in data.sections
+        section(class="service")
+            img(src=section.img)
+            div(class="service-item")
+                h2(class="service-title") #{section.title}
+                p #{section.description}
+                p #{section.subdescription}
+                each item in section.items 
+                    li #{item.item}
+                each contact in section.contacts  
+                    p #{contact.contact}
+        +section("cards")
+            div(class="cards") 
+                include ../../mixins/components.pug
+                +flipCard("Rentals", "/assets/img/flipcards/msoc-22-scaled.jpeg")
+                    img(src="/assets/img/icons/tie-icon.svg")
+                    p We provide Textbook Rentals, Board Game Rentals, Calculator Rentals, and equipment for our clubs and members!
```

Comment:
can you add all the text like "We provide Textbook Rentals...." to the mathsoc-office.json file?